hack/make.ps1

```diff
@@ -94,6 +94,8 @@ param(
 $ErrorActionPreference = "Stop"
 $ProgressPreference = "SilentlyContinue"
 $pushed=$False  # To restore the directory if we have temporarily pushed to one.
+Set-Variable GOTESTSUM_COMMIT -option Constant -value 'v0.3.5'
```

Comment:
Is this variable still used? (I see it's also declared in the Dockerfile)?

Reply:
Removed